.github/workflows/docs.yml

```diff
@@ -0,0 +1,35 @@
+name: Build and Deploy docs
+
+on:
+  push:
+    branches: [master]
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Crystal
+        uses: oprypin/install-crystal@v1
+
+      - name: Download source
+        uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+
+      - name: Install dependencies
+        run: shards install
```

Comment:
Pretty sure Ameba doesn't have any dependencies.

Reply:
Doesn't hurt either, in case one day it would :)